src/components/panel/demoConfigProvider/index.html

```diff
@@ -0,0 +1,51 @@
+<div ng-controller="PanelProviderCtrl as ctrl" ng-cloak>
+
+  <md-content layout-padding>
+
+    <div layout="row" layout-align="space-between center" flex>
+      <md-button
+        class="md-fab md-primary"
+        aria-label="Navigation"
+        ng-click="ctrl.showMenu($event, ctrl.navigation)">
+        <md-icon md-svg-icon="img/icons/menu.svg"></md-icon>
+      </md-button>
+      <md-button
+        class="md-fab md-accent"
+        aria-label="Favorites"
+        ng-click="ctrl.showMenu($event, ctrl.favorites)">
+        <md-icon md-svg-icon="img/icons/favorite.svg"></md-icon>
+      </md-button>
+      <md-button
+        class="md-fab md-background"
+        aria-label="More"
+        ng-click="ctrl.showMenu($event, ctrl.more)">
+        <md-icon md-svg-icon="img/icons/more_vert.svg"></md-icon>
+      </md-button>
+    </div>
+
+    <p>
+      Configurations for upcoming panel elements can be created at the
+      <code>.config</code> level of your Angular application. To create a
+      configuration object, use the <code>$mdPanelConfigProvider</code>
+      dependency within a <code>.config</code> method and call
+      <code>$mdPanelConfigProvider.createConfig</code> with an object containing
+      a configuration name and options as it's argument. This object will be
+      stored for you so that you can retrieve it by using the
+      <code>$mdPanel.getConfig</code> method with the name of the configuration
+      object that you created as it's argument.
+    </p>
+    <p>
+      The configuration object takes all of the options found within the
+      <code>$mdPanel.create</code> method; however, it does not make sense to
```

Comment:
Rather it's impossible to include options related to user interaction, panel positioning, or panel animations.